rye/src/cli/init.rs

```diff
@@ -700,7 +700,8 @@ fn import_requirements_file(
     data.requirements.iter().for_each(|x| {
         requirements
             .entry(x.requirement.name.to_string())
-            .or_insert(format_requirement(&x.requirement).to_string());
+            // todo: will to_string this work?
+            .or_insert(x.requirement.to_string());
```

Comment:
tested - the missing format_requirement here messes up `${VARIABLES}` in requirements files. It's not called in the PR because of the pep440 version divergence. One could depend on both new and old, any solution feels gross here.